### ISSUE TITLE
Update Release Checklist with PAT Reminder

### ISSUE THREAD

**Issue by tsande16** [eclipse-pass/main#1079]
# What? 

Update the release checklist to include a reminder for the release manager to generate a Personal Access Token (PAT) before starting the release.

# Why?

At the moment a PAT is required for performing the release. Having this reminder will prevent a release from failing by not having one generated. 

# How?

Add reminder to the checklist.

# Acceptance Criteria 

A reminder to generate the PAT is on the release checklist.

**Comment by rpoet-jh** [comment]
@tsande16 this is done, right?

**Comment by tsande16** [comment]
Correct. Looks like I merged it with another set of updates as well in PR #1096, ticket #1056
